src/app/app-interactive-map/app-interactive-map.component.ts

```diff
@@ -41,27 +40,41 @@ const fluxFilter = objectFilter((key, value) => Math.abs(value) > 1e-7);
 export class AppInteractiveMapComponent implements OnInit, AfterViewInit {
   private builderSubject = new Subject<escher.BuilderObject>();
   public map: Observable<escher.PathwayMap>;
-  public model: Observable<Cobra.Model>;
+  public model: Cobra.Model;
   public loading = true;
+  private card: Card;
```

Comment:
I still think the card belongs in the store and should be selected, instead of copied to the component

Reply:
But, I think  to use it inside of the controller I have to do it like this, but let me take a look about it.
Because if you want to use in the template you can just subscribe and then use the async, but in the controller we are not allow to do it.